lib/src/Uncertainty/Distribution/ComposedDistribution.cxx

```diff
@@ -148,15 +148,20 @@ String ComposedDistribution::__repr__() const
 
 String ComposedDistribution::__str__(const String & ) const
 {
-  OSS oss;
-  oss << getClassName() << "(";
-  String separator("");
+  OSS oss(false);
+  oss << "- " << getClassName() << "\n";
+  oss << "- " << "dimension=" << getDimension() << "\n";
+  oss << "- " << "description= " << description_ << "\n";
+  if (getDimension() > 1) oss << "- copula=" << copula_.__str__() << "\n";
+  oss << "\n";
+  oss << "| Index | Variable | Distribution |\n";
+  oss << "|-------|----------|--------------|\n";
```

Comment:
Looks like the distribution column is not wide enough on most cases

Reply:
This is true. This does not prevent the Markdown format from being valid. Moreover, it does not prevent a Markdown editor to format the string correctly. I do not think that setting a default column width to a large value, say 60 for example, can be satisfactory. I do not know how this limitation can be cleanly solved.

Reply:
the width can be computed as the maximum of the width of each line

Reply:
Fixed at 8cd1e8e1c